### PR TITLE
Updated the resiliency status page for v1.4.0

### PR DESCRIPTION
**Pull request by clintongormley**

### CODE REVIEW

**Comment by clintongormley** [comment]
@bleskes @kimchy Please could you review

File: docs/resiliency/index.asciidoc [hunk 1]
```diff
@@ -56,6 +56,72 @@ If you encounter an issue, https://github.com/elasticsearch/elasticsearch/issues
...
+[float]
+=== OOM resiliency (STATUS: ONGOING)
+
+The family of circuit breakers has greatly reduced the occurrence of OOM
+exceptions, but it is still possible to cause a node to run out of heap
+space.  The following issues have been identified:
+
+* Set a hard limit on `from`/`size` parameters {GIT}9311[#9311]. (STATUS: ONGOING)
+* Prevent combinatorial explosion in aggregations from causing OOM {GIT}8081[#8081]. (STATUS: ONGOING)
+* Add the byte size of each result to the request circuit breaker {GIT}9310[#9310]. (STATUS: ONGOING)
```
Comment (kimchy) [inline docs/resiliency/index.asciidoc]: of each hit? slightly more explanatory

**Comment by kimchy** [comment]
LGTM, I am missing more work done in 1.5 (or I just can't find it), like using the `.si` file as the unique id to check for segments on recovery, and not recovering from nodes with known bad versions like ones that have the compression bug

**Comment by clintongormley** [comment]
OK merging - I'll address the 1.5 changes in a different PR, don't want to delay this one any more

**Pull request closed without merging**